lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,643 @@
+                                                       data-refund-info="<%- gettext('You will be refunded the amount you paid.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% }
+                                            } else {
+                                                if (!model.is_course_blocked) { %>
+                                                    <a href="#unenroll-modal" class="action action-unenroll"
+                                                       rel="leanModal" data-course-id="<%- model.course_key %>"
+                                                       data-course-number="<%- model.course_number %>"
+                                                       data-course-name="<%- model.course_name %>"
+                                                       data-dashboard-index="<%- dashboard_index %>"
+                                                       data-cert-name-long="<%- settings.cert_name_long %>"
+                                                       data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                       data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% } else { %>
+                                                    <a class="action action-unenroll is-disabled"
+                                                       data-course-id="<%- model.course_key %>"
+                                                       data-course-number="<%- model.course_number %>"
+                                                       data-course-name="<%- model.course_name %>"
+                                                       data-dashboard-index="<%- dashboard_index %>"
+                                                       data-cert-name-long="<%- settings.cert_name_long %>"
+                                                       data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                       data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% }
```

Comment:
@AlasdairSwan There will be a lot of repetitive code in the view if I move it there. The template has access to calculated values which have to be recalculated on the server side for each course enrolment, which will introduce duplicate code. Besides, this is already a simpler version of the mako template I ported from. Thoughts?